.github/workflows/build_tests.yaml

```diff
@@ -49,6 +49,12 @@ jobs:
       run: |
         gcloud config set compute/zone us-east4-a
         gcloud config get compute/zone
+    - name: Install kjob
+      run: |
+        git clone https://github.com/kubernetes-sigs/kueue.git
+        cd kueue/cmd/experimental/kjobctl
+        make kubectl-kjob
+        mv ./bin/kubectl-kjob /usr/local/bin/kubectl-kjob
```

Comment:
Does it work without calling `make install`?

Reply:
[make install](https://github.com/kubernetes-sigs/kueue/blob/615a81ede077078a86d1f4881a2029b212a669a5/cmd/experimental/kjobctl/Makefile#L160)

consists of: ```(KUSTOMIZE) build config/crd | $(KUBECTL) apply --server-side -f -``` commands
which i copied and executed in L160 of kjob.py during cluster creation. We only apply make kubectl-kjob as specified in [instruction](https://github.com/kubernetes-sigs/kueue/blob/main/cmd/experimental/kjobctl/docs/installation.md), make install is applied on cluster during creation.